app/views/users/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<div>
+  <h2> Name: <%= @user.name %> </h2>
+  <h2> Bio: <%= @user.bio %> </h2>
+  <%@posts.each_with_index do |post, index|%>
+    <h4>post #<%= index + 1%></h4>
+    <h2>Title: <%=post.title%></h2>
+    <p>Description: <%=post.text%></p>
+    <%= link_to "Show all posts", "/users/#{@user.id}/posts"%>
```

Comment:
- [ ] @Fayob great work implementing the changes requested as you can see the show all post button appears in all shown posts kindly move it outside the posts loop so that it can appear only once at the bottom
![image](https://user-images.githubusercontent.com/61408860/214809224-294e1133-52fd-4643-9fd8-4d95bd53e2e1.png)